blueocean-personalization/package.json

```diff
@@ -26,6 +26,7 @@
     "enzyme": "^2.2.0",
     "eslint": "2.8.0",
     "eslint-plugin-react": "^5.0.1",
+    "eslint-to-editorconfig": "1.2.0",
```

Comment:
This only generates the editorconfig unfortunately, I just added it to make sure the two were in sync with each other. It more or less matches the one from dashboard fine.